mapdata/WarcraftLegacies/AbilityData/Skin/808726849.json

```diff
@@ -0,0 +1,109 @@
+{
+  "OldId": 1650739521,
+  "NewId": 808726849,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(DUMMY frostwolf)"
+    },
+    {
+      "Id": 1953653089,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNOtherBarrel.blp"
+    },
+    {
+      "Id": 2020631137,
+      "Value": 2
+    },
+    {
+      "Id": 2037408353,
+      "Value": 2
+    },
+    {
+      "Id": 2020635233,
+      "Value": 2
+    },
+    {
+      "Id": 1918988897,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNOtherBarrel.blp"
+    },
+    {
+      "Id": 1952805473,
+      "Type": 3,
+      "Value": "Learn Rounds for Everyone - [|cffffcc00Level %d|r]"
```

Comment:
Is this ability actually used? Can't find it. Needs a delete if not.

Reply:
isht this chen's ability?

Reply:
This is a new file that you're adding with this PR.

Reply:
huh, then its useless, can be deleted

Reply:
You'll need to delete in on your end.